Documentation/Storage-Configuration/NFS/nfs-security.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Security
+---
+
+## User ID management
+
+!!! attention
+    This feature is experimental and will not support upgrades to future versions.
+
+!!! attention
+    Some configurations of this feature may break the ability to
+    [mount NFS storage to pods via PVCs](./nfs-csi-driver.md#attaching-an-export-to-a-pod).
+
+### ID management via SSSD
+
+[SSSD](https://sssd.io) is the System Security Services Daemon. It can be used to provide user ID
+management from a  number of sources including LDAP, Active Directory, and FreeIPA. Currently, only
+LDAP has been tested.
+
+!!! attention
+    The Ceph container image must have the `sssd-client` package installed to support SSSD. This
+    package is included in `quay.io/ceph/ceph` in v17.2.4 and newer. For older Ceph versions you may
+    build your own Ceph image which adds `RUN yum install sssd-client && yum clean all`.
+
+#### **SSSD configuration**
+
+SSSD requires a configuration file in order to configure its connection to the user ID management
+system (e.g., LDAP). The file follows the `sssd.conf` format documented in its
+[man pages](https://linux.die.net/man/5/sssd.conf).
+
+Methods of providing the configuration file are documented in the
+[NFS CRD security section](../../CRDs/ceph-nfs-crd.md#security).
+
+The SSSD configuration file may be omitted from the CephNFS spec if desired. In this case, Rook will
+not set the `/etc/sssd/sssd.conf` in any way. This allows you to manage the `sssd.conf` file
+yourself however you wish. For example, you may build it into your custom Ceph container image, or
+use the [Vault agent injector](https://www.vaultproject.io/docs/platform/k8s/injector) to securely
+add the file via annotations on the CephNFS spec (passed to the NFS server pods).
```

Comment:
Does this section seem adequate to explain how to configure things?
1. refer to CRD docs
2. some more info if you want to specify your own

Reply:
Yes this is helpful to have the pointers for how they might choose to do it.